src/uct/api/v2/uct_v2.h

```diff
@@ -843,7 +843,12 @@ typedef enum {
      * packed key by @ref uct_md_mkey_pack_v2 with
      * @ref UCT_MD_MKEY_PACK_FLAG_INVALIDATE flag.
      */
-    UCT_MD_FLAG_INVALIDATE_AMO = UCS_BIT(12)
+    UCT_MD_FLAG_INVALIDATE_AMO = UCS_BIT(12),
+
+    /**
+     * Memory domain supports device memory registration.
+     */
+    UCT_MD_FLAG_REG_MEMIC_MR   = UCS_BIT(13)
```

Comment:
```
/**
 * The memory domain supports allocating device memory.
 */
UCT_MD_FLAG_ALLOC_DEVICE_MEMORY    = UCS_BIT(13)
```

Reply:
Agree - memic is something very Mellanox specific. Let's use generic terms.